src/app/containers/Ad/Amp/index.jsx

```diff
@@ -91,40 +115,54 @@ export const AMP_ACCESS_FETCH = service => {
 };
 
 // eslint-disable-next-line react/prop-types
-const AmpAd = ({ service }) => {
+const AmpAd = () => {
+  const { ads, dir, script, service } = useContext(ServiceContext);
+  const label = pathOr('Advertisement', ['advertisementLabel'], ads);
+
   return (
-    <StyledWrapper>
-      <Helmet>
-        {AMP_ADS_JS}
-        {AMP_ACCESS_JS}
-        {AMP_ACCESS_FETCH(service)}
-      </Helmet>
-      <div
-        amp-access="toggles.ads.enabled AND geoIp.advertiseCombined"
-        amp-access-hide="true"
-      >
-        <StyledAd>
-          <amp-ad {...ampAdPropsMobile({ service })}>
-            <amp-img
-              placeholder
-              width="320"
-              height="50"
-              src="https://via.placeholder.com/320x50"
-              layout="responsive"
-            />
-          </amp-ad>
-          <amp-ad {...ampAdPropsDesktop({ service })}>
-            <amp-img
-              placeholder
-              width="970"
-              height="250"
-              src="https://via.placeholder.com/970x250"
-              layout="responsive"
-            />
-          </amp-ad>
-        </StyledAd>
-      </div>
-    </StyledWrapper>
+    <FullWidthWrapper>
+      <StyledWrapper>
+        <Helmet>
+          {AMP_ADS_JS}
+          {AMP_ACCESS_JS}
+          {AMP_ACCESS_FETCH(service)}
+        </Helmet>
+        <div
+          amp-access="toggles.ads.enabled AND geoIp.advertiseCombined"
+          amp-access-hide="true"
+        >
+          <StyledAd>
+            <StyledLink
+              href={LABEL_LINK}
+              script={script}
+              service={service}
+              dir={dir}
+              tabIndex="-1"
```

Comment:
I believe you can add it directly to the styled component as an attribute https://github.com/bbc/simorgh/pull/6488/files#r427886995